### DISCUSSION TITLE
SuperSlicer encountered an error (uploading to print host)

### DISCUSSION THREAD

**Discussion by troypiggo** [supermerill/SuperSlicer#4292] [General]
In the printer settings the test works fine "Connection to Klipper works correctly".  The URL is http://mainsail.lan and host type Klipper.  I'm pretty sure in previous versions of SS and Mainsail/Klipper this works, but I've recently reinstalled everything and now this feature is not working.  When I try to upload, I get this error.  Any tips?

```
Error uploading to print host:
Send failed since rewinding of the data stream failed:
Cannot rewind mime/post data
[Error 65]
```

**Comment by darco415** [comment]
what ver of SS are you on? Did you try just entering the IP instead? Works here on .10 with IP

**Reply by troypiggo** [reply]
Thanks for the suggestions, mate.  SS version is the latest Windows one.
Last night I tried just the IP.  It wouldn't even pass the test, whereas using the http URL did.  And just the IP also didn't upload, but the error message was different.
I saw somewhere else a suggestion to change a setting from 1 to 0 in the config ini file.  Tried that too, didn't work.
Sorry the above is vague (not posting the actual error message) - I'm doing it from memory as I'm at work.
